ui/src/state/PatronCurrentBorrowingRequests/actions.js

```diff
@@ -0,0 +1,44 @@
+import { IS_LOADING, SUCCESS, HAS_ERROR } from './types';
+import { invenioConfig } from '@config';
+import { illBorrowingRequest as BorrowingRequestApi } from '@api';
+import { sendErrorNotification } from '@components/Notifications';
+
+const selectQuery = (patronPid, page = 1, size) => {
+  const statuses = invenioConfig.illBorrowingRequests.pendingStatuses.concat(
```

Comment:
what do you think about using [_.difference](https://lodash.com/docs/4.17.15#difference)?
```
import _difference from 'lodash/difference';
const illConfig = invenioConfig.illBorrowingRequests;
const statuses = _difference(illConfig.orderedValidStatuses, illConfig.completedStatuses);
```
to be tested!